src/inverseRobot.js

```diff
@@ -23,8 +23,30 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  let count = 0;
+  const robValues = Object.values(robot).reverse();
+  const robKeys = Object.keys(robot).reverse();
+  const obj = {};
+
+  for (let i = 0; i < robValues.length; i++) {
+    for (let j = i + 1; j < robValues.length; j++) {
+      if (robValues[i] === robValues[j]) {
+        count++;
+      }
+    }
+  }
+
+  if (count > 0) {
+    return null;
+  }
+
+  for (let i = 0; i < robKeys.length; i++) {
+    obj[robValues[i]] = robKeys[i];
+  }
```

Comment:
There is a way to do it easier, without 2 loops, try using `for ... in` loop, and proper check from [Checklist's](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md) 2nd item